uilib/src/components/Text/getTextStyle.ts

```diff
@@ -0,0 +1,117 @@
+export type TextTypes =
+  | "h1"
+  | "h2"
+  | "h3"
+  | "highlight"
+  | "emphasis"
+  | "body"
+  | "cta"
+  | "link"
+  | "tiny"
+  | "subTitle"
+  | "navigation"
+  | "tag";
+
+export default function getTextStyle({
+  type,
+  bracket,
+}: {
+  type: TextTypes;
+  bracket?: boolean;
+}): {
+  fontFamily: string;
+  fontSize: number;
+  lineHeight: number;
+  fontWeight?: number;
+  textDecoration?: string;
+  paddingTop?: number;
+} {
+  switch (type) {
+    case "h1":
+      return {
+        fontFamily: "Alpha",
+        fontSize: 36,
+        lineHeight: 43.2,
+        paddingTop: bracket ? 15 : 0,
```

Comment:
I think this is causing the following: 
![Uploading Screenshot 2021-08-16 at 17.52.00.png…]()